LuckParser/Controllers/HTMLHelper.cs

```diff
@@ -749,69 +750,74 @@ public static void WriteCombatReplayInterface(StreamWriter sw, Tuple<int,int> ca
         public static void WriteCombatReplayScript(StreamWriter sw, ParsedLog log, Tuple<int,int> canvasSize, CombatReplayMap map, int pollingRate)
         {
             sw.WriteLine("<script>");
+            sw.WriteLine(Properties.Resources.combatreplay_js);
+            sw.WriteLine("</script>");
+
+            Dictionary<string, object> options = new Dictionary<string, object>
+            {
+                { "inch", map.GetInch() },
+                { "pollingRate", pollingRate },
+                { "mapLink", map.Link }
+            };
+
+            string actors = "";
+            int count = 0;
+            foreach (Player p in log.PlayerList)
             {
-                string replayScript = Properties.Resources.combatreplay_js;
-                replayScript = replayScript.Replace("'${inch}'", map.GetInch().ToString());
-                replayScript = replayScript.Replace("'${pollingRate}'", pollingRate.ToString());
-                replayScript = replayScript.Replace("'${mapLink}'", map.Link);
-                string actors = "";
-                CombatReplay replay;
```

Comment:
most of the following lines is just whitespace changes, you should click "**Diff settings**" and enable "**Hide whitespace changes**" here.